1-Draft/RFCXXXX-Obsoletion-and-Deprecation.md

```diff
@@ -185,32 +201,26 @@ WARNING: property1 is obsolete. Use property2.
 
 ```
 
-This section provides suggestions on global behaviors as well as overriding these behaviors.
-
-### Displaying obsolete behavior
-
-Currently, all warnings are displayed based on the global preference variable `WarningPreference`.
-Additional levels of granularity would be useful so different warnings may be controlled, much the same way that trap/catch works in PowerShell today.
-While it is the case that you can use `-WarningAction SilentlyContinue` with cmdlets today, it will affect _all_ warnings.
-In some cases, you may not want to suppress obsolete warnings while suppressing operational warnings (and vice-versa).
-
-With regard to types, when an obsolete type or member is used, we will emit a warning:
-
-- when a script is _parsed_
-  - This means that when a script is executed or dot-sourced a warning may be emitted
-- when a module is imported (a special case of parsing)
-- when an obsolete type is _used_ in an interactive session
-
 There is some asymmetry with behavior here, as Cmdlet obsolescence warnings occur when the cmdlet it _used_.
 However, most PowerShell users do not notice the difference between our parsing phase and execution phase.
-Further, we don't want to emit on type use as it could result in 1000s of warnings
 
 ---
 **NOTE:**
-We should also consider altering our current behavior for cmdlets to emit a warning only at parse time rather than run-time.
+We should consider altering our current behavior for cmdlets to emit a warning only at parse time rather than run-time.
 
 ---
 
+## Displaying and Overriding default behaviors
+
+This section provides proposals on global behaviors as well as overriding these behaviors.
+
+### Displaying obsolete behavior
+
+Currently, all warnings are displayed based on the global preference variable `WarningPreference`.
+Additional levels of granularity would be useful so different warnings may be controlled, much the same way that trap/catch works in PowerShell today.
+While it is the case that you can use `-WarningAction SilentlyContinue` with cmdlets today to suppress obsolescence warnings, this will affect _all_ warnings.
+In some cases, you may not want to suppress obsolete warnings while suppressing operational warnings (and vice-versa).
+
 #### ObsoleteWarningPreference
 
 This is a new ubiquitous variable will dictate the global behavior of PowerShell when an item has been made obsolete.
```

Comment:
I don't think that that is a proper use of the word "ubiquitous". Do you mean that the variable is not scoped? Also, "variable will dictate" s/b "variable which will dictate".

Reply:
Usually these are called automatic variables, to my memory.